datalad/interface/tests/test_base.py

```diff
@@ -150,3 +154,29 @@ def test_nadict():
     d = nadict({1: 2})
     eq_(d[1], 2)
     eq_(str(d[2]), NA_STRING)
+
+
+@with_tempfile(mkdir=True)
+def test_status_custom_summary_no_repeats(path):
+    from datalad.api import Dataset
+    from datalad.cmd import Runner
+    from datalad.core.local.status import Status
+
+    # This regression test depends on the command having a custom summary
+    # renderer that renders output. status() having this method doesn't
```

Comment:
If we keep this test: "custom summary renderer that renders output" sounds silly.  I should rephrase it.  I was trying to make the point that this test depends on the custom summary renderer outputting *something*, which isn't a given (e.g., the call in this test wouldn't have any output if annex data weren't added).